spec/features/visitor_can_view_items_spec.rb

```diff
@@ -1,28 +1,30 @@
 require "rails_helper"
 
-feature "View items" do
-  let!(:item_one) {Item.create(name: "Canon 7d",
-                                description: "The best camera ever (NOT)",
-                                price: 15)}
+RSpec.describe "VisitorCanViewItems", type: :feature do
+  feature "View items" do
+    let!(:item_one) {Item.create(name: "Canon 7d",
+                                  description: "The best camera ever (NOT)",
+                                  price: 15)}
 
-  let!(:item_two) {Item.create(name: "Sony a7r",
-                                description: "Robbies camera",
-                                price: 12)}
+    let!(:item_two) {Item.create(name: "Sony a7r",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.